cli/morphir-elm-develop.js

```diff
@@ -28,33 +28,39 @@ const wrap = fn => (...args) => fn(...args).catch(args[2])
 const webDir = path.join(__dirname, 'web')
 const indexHtml = path.join(webDir, 'index.html')
 
+const createSimpleGetJsonApi = (filePath) => {
+  app.get('/server/' + filePath, wrap(async (req, res, next) => {
+    const jsonPath = path.join(program.opts().projectDir, filePath)
+    const jsonContent = await readFile(jsonPath)
+    res.send(JSON.parse(jsonContent.toString()))
+  }))
+}
+
+app.use(express.static(webDir))
+app.use(express.json());
+
 app.get('/', (req, res) => {
   res.sendFile(indexHtml)
 })
 
-app.use(express.static(webDir))
-app.use(express.json());
+createSimpleGetJsonApi('morphir.json')
+createSimpleGetJsonApi('morphir-ir.json')
+createSimpleGetJsonApi('morphir-tests.json')
 
-app.get('/server/morphir.json', wrap(async (req, res, next) => {
-  const morphirJsonPath = path.join(program.opts().projectDir, 'morphir.json')
-  const morphirJsonContent = await readFile(morphirJsonPath)
-  const morphirJson = JSON.parse(morphirJsonContent.toString())
-  res.send(morphirJson)
+app.get('/server/attributeconfs/:attributename', wrap(async (req, res, next) => {
+  const attributeName = req.params.attributename.toString()
+  const configPath = path.join(program.opts().projectDir, attributeName + '-attribute.conf.json')
```

Comment:
Can you start creating a user guide as an .md file in the docs folder that describes how to define a custom attribute? This should include the file naming conventions described here and updated as we develop the feature.